package/gluon-mesh-vpn-wireguard/src/gluon-mesh-vpn-key-translate.c

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2022 Jan-Niklas Burfeind <gluon@aiyionpri.me>
+// SPDX-License-Identifier: BSD-2-Clause
+// SPDX-FileContributor: read_key() by Matthias Schiffer <mschiffer@universe-factory.net>
+
+#include <libubox/utils.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define FASTD_KEY_LEN_HEX 64
+
+static inline void invalid_key(void) {
```

Comment:
Using explicit `inline` outside of header files is usually unnecessary, the compiler can figure out by itself what to inline.

Reply:
resolved in e854c5635706ebf18bc9568abca2389e60df38b8